release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1566,6 +1566,12 @@ sourceStrategy:
 
 * Previously, if a `configmap` that the router deployment depends on is not created, then the router deployment does not progress. With this update, the cluster Operator reports `ingress progressing=true` if the default ingress controller deployment is progressing. This results in users debugging issues with the ingress controller by using the command `oc get co`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2066560[*BZ#2066560*])
 
+* Previously, when an incorrectly created network policy was added to the OVN-Kubernetes cache, it would cause the OVN-Kubernetes leader to enter `crashloopbackoff` status. With this update, OVN-Kubernetes leader does not enter `crashloopbackoff` status by skipping deleting nil policies. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2091238[*BZ#2091238*])
+
+* Previously, recreating an EgressIP pod with the same namespace or name within 60 seconds of deleting an older one with the same namespace or name causes the wrong SNAT to be configured. As a result, packets could go out with nodeIP instead of EgressIP SNAT. With this update, traffic leaves the pod with EgressIP instead of nodeIP. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097243[*BZ#2097243*]).
+
+* Previously, older Access Control Lists (ACL)s with `arp` produced errors `unexpectedly found multiple equivalent ACLs (arp v/s arp||nd)` due to a change in ACL from `arp` to `arp II nd`. This prevented network policies from being created properly. With this update, older ACLs with just the `arp` match have been removed so that only ACLs with the new `arp II nd` match exist so that network policies can be created correctly and no errors will be observed on `ovnkube-master`. NOTE: This effects customers upgrading into 4.8.14, 4.9.32, 4.10.13 or higher from older versions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2095852[*BZ#2095852*]).
```

Comment:
s/errors `unexpectedly found multiple equivalent ACLs (arp v/s arp||nd)`/`unexpectedly found multiple equivalent ACLs (arp v/s arp||nd)` errors
s/in ACL/in the ACL